README.md

```diff
@@ -1,5 +1,12 @@
 # Alfresco Digital Business Platform Deployment
 
+The Alfresco Digital Business Platform can be deployed to different platforms: AWS or locally.
```

Comment:
Should we leave it slightly more open here?
> .. can be deployed to different **environments** such as AWS or locally

Reply:
I like that.

Reply:
https://youtu.be/NGc7izPYk6A?t=1m30s :)